servicemesh-install/servicemesh-install.adoc

```diff
@@ -72,3 +72,8 @@ include::servicemesh-install/topics/install-remove.adoc[leveloffset=+2]
 == Upgrading {ProductName}
 
 include::servicemesh-install/topics/install-upgrade.adoc[leveloffset=+2]
+
+[[appendix_chapter_9]]
+== 3scale Istio Adapter
```

Comment:
Label the Appendix here (not in the linked file) so that the TOC generates correctly.

[[appendix_A]]
== Appendix: 3scale Istio Adapter